.github/workflows/generate-code-owners.yml

```diff
@@ -0,0 +1,32 @@
+name: Update CODEOWNERs
+
+on:
+  schedule:
+    # https://crontab.guru/#5_8_*_*_1
+    - cron: "5 8 * * 1"
```

Comment:
Should we consider running it on every push to `main` - that means some new extension was added and file may need to be regenerated.

Reply:
made it run when a package.json has changed